multicluster/test/e2e/framework.go

```diff
@@ -45,6 +45,10 @@ const (
 	leaderCluster             string = "leader-cluster"
 	serviceExportYML          string = "serviceexport.yml"
 
+	testServerPod           string = "test-nginx-pod"
+	gatewayNodeClientSuffix string = "gateway-client"
+	regularNodeClientSuffix string = "regular-client"
```

Comment:
Please check github check results and fix it, Golangci-lint shows `randName` is unused